frontend/src/scenes/notebooks/Nodes/NotebookNodeBacklink.tsx

```diff
@@ -0,0 +1,92 @@
+import { mergeAttributes, Node, NodeViewProps } from '@tiptap/core'
+import { NodeViewWrapper, ReactNodeViewRenderer } from '@tiptap/react'
+import { InsightModel, NotebookNodeType } from '~/types'
+import { Link } from '@posthog/lemon-ui'
+import { IconGauge, IconBarChart, IconFlag, IconExperiment, IconLive, IconPerson, IconCohort } from 'lib/lemon-ui/icons'
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { urls } from 'scenes/urls'
+import { useActions, useValues } from 'kea'
+import { notebookSidebarLogic } from '../Notebook/notebookSidebarLogic'
+import { notebookLogic } from '../Notebook/notebookLogic'
+import clsx from 'clsx'
+
+const ICON_MAP = {
+    dashboards: <IconGauge />,
+    insights: <IconBarChart />,
+    feature_flags: <IconFlag />,
+    experiments: <IconExperiment />,
+    events: <IconLive width="1em" height="1em" />,
+    persons: <IconPerson />,
+    cohorts: <IconCohort />,
+}
+
+const Component = (props: NodeViewProps): JSX.Element => {
+    const { shortId } = useValues(notebookLogic)
+    const { notebookLinkClicked } = useActions(notebookSidebarLogic)
+
+    const type: TaxonomicFilterGroupType = props.node.attrs.type
+    const href: string | undefined = backlinkHref(props.node.attrs.id, type)
+    const title: string = props.node.attrs.title
+
+    const isViewing = window.location.pathname === href
+
+    console.log(href)
+
+    return (
+        <NodeViewWrapper as="span" class={clsx('backlink', isViewing && 'active', props.selected && 'selected')}>
+            <Link to={href} onClick={() => notebookLinkClicked(shortId)} target={undefined} className="space-x-1">
+                <span>{ICON_MAP[type]}</span>
+                <span className="label">{title}</span>
+            </Link>
+        </NodeViewWrapper>
```

Comment:
Following our styleguide - any non-utility classes should be BEM style so `Backlink` and `Backlink__label` `Backlink--active` etc.